### ISSUE TITLE
Document that 0.13.5 is required and how to download

### ISSUE THREAD

**Issue by costrouc** [Quansight/qhub-cloud#348]
The docs should detail how to install terraform and a note that a certain version is required.

1. Download the binary with url
2. Add to `~/.local/bin`
3. Add to your path `export PATH=$HOME/.local/bin:$PATH`if not already in your path

**Comment by laisbsc** [comment]
The instructions to manually install Terraform version 0.13.5 are [here](https://github.com/Quansight/qhub-cloud/blob/dev/docs/source/02_get_started/01_installation.md#install-terraform), on the latest merged version of the docs (`dev` branch). 
I can make it a bit more detailed (such as your instructions in the issue).

In addition, we will have a Troubleshooting section and one of the topics is how to Downgrade an installed version of Terraform. Since the page is not pushed to GH just yet, I copied the docs text below.

---

#### Downgrade an installed Terraform version

QHub is currently ONLY compatible with Terraform version 0.13.5. To check your installed version on terminal type `terrafrom -v`.

If your local version is more recent than the recommended, follow the steps:


On terminal type `which terraform` to find out where the binary installation file is located.
The output will be a filepath, such as: `/usr/bin/terraform`
Then, remove the binary file with:
```shell
rm -r /usr/bin/terraform
```
> `/usr/bin/terraform` corresponds to the location of the installed binary file.

Next, download the binary for the compatible version with:
```shell
wget https://releases.hashicorp.com/terraform/0.13.5/
```
Unzip the file, and move it to the same location as the previous one:
```shell
unzip terraform_0.13.5
mv ~/Downloads/terraform /usr/bin/
```
Finally, check if the correct version was installed with `terraform -v`, the output should be `Terraform v0.13.5`.

**Comment by laisbsc** [comment]
Solved by PR #356.
